cmd/codecrafters/main.go

```diff
@@ -23,11 +23,16 @@ USAGE
   $ codecrafters [command]
 
 EXAMPLES
-  $ codecrafters test     # Run tests without committing changes
-  $ codecrafters submit   # Commit changes & submit to move to next step
+  $ codecrafters test              # Run tests without committing changes
+  $ codecrafters test --previous   # Run tests for the current stage and all previous stages in ascending order
+  $ codecrafters submit            # Commit changes & submit to move to next step
 
 COMMANDS
   test:        Run tests without committing changes
+
+               Flags for test:
+               --previous   Run tests for the current stage and all previous stages in ascending order
```

Comment:
Let's remove the `flags` section from here for now - will think about how to do proper flags per command. We need a better system for this - something like `codecrafters test --help`.